virt/virt-4-11-release-notes.adoc

```diff
@@ -162,91 +160,84 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
+
+* If you use Red Hat Ceph Storage or Red Hat OpenShift Data Foundation Storage, cloning more than 100 VMs at once might fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1989527[*BZ#1989527*])
+** As a workaround, you can perform a host-assisted copy by setting `spec.cloneStrategy: copy` in the storage profile manifest. For example:
 +
 [source,yaml]
 ----
-apiVersion: kubevirt.io/v1alpha3
-kind: VirtualMachine
+apiVersion: cdi.kubevirt.io/v1beta1
+kind: StorageProfile
 metadata:
-  annotations:
-...
+  name: <provisioner_class>
+#   ...
+spec:
+  claimPropertySets:
+  - accessModes:
+    - ReadWriteOnce
+    volumeMode: Filesystem
+  cloneStrategy: copy <1>
+status:
+  provisioner: <provisioner>
+  storageClass: <provisioner_class>
 ----
-+
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1979114[*BZ#1979114*])
-+
-** When connecting to the VNC console by using the {VirtProductName} web console, the VNC console always fails to respond.
-+
-*** As a workaround, create the virtual machine from the CLI or upgrade to {VirtProductName} 4.8.
-+
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
+<1> The default cloning method set as `copy`.
+
+* In some instances, multiple virtual machines can mount the same PVC in read-write mode, which might result in data corruption. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1992753[*BZ#1992753*])
+** As a workaround, avoid using a single PVC in read-write mode with multiple VMs.
+
+* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2026733[*BZ#2026733*])
+** As a workaround, xref:../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].
+
+* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037611[*BZ#2037611*])
+** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
 
-//Known issues discovered after 4.9 release
+* If you configure the `HyperConverged` custom resource (CR) to enable mediated devices before drivers are installed, enablement of mediated devices does not occur. This issue can be triggered by updates. For example, if `virt-handler` is updated before `daemonset`, which installs NVIDIA drivers, then nodes cannot provide virtual machine GPUs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2046298[*BZ#2046298*])
```

Comment:
I'd rephrase "enablement of mediated devices does not occur" to something less passive. Maybe:

```suggestion
* If you configure the `HyperConverged` custom resource (CR) to enable mediated devices before drivers are installed, the new device configuration does not take effect. This issue can be triggered by updates. For example, if `virt-handler` is updated before `daemonset`, which installs NVIDIA drivers, then nodes cannot provide virtual machine GPUs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2046298[*BZ#2046298*])
```